meta/main.yml

```diff
@@ -18,10 +18,21 @@ galaxy_info:
   platforms:
     - name: Debian
       versions:
+        - buster
         # Kali linux isn't an option here, but it is based on
         # Debian Testing:
         # https://www.kali.org/docs/policy/kali-linux-relationship-with-debian
-        - bullseye
+        #
+        # Unfortunately, the Debian Bullseye deb packages available at
+        # https://github.com/PowerShell/PowerShell do not install on a
+        # recent Debian Bullseye instance due to broken dependencies.
+        # Kali Linux provides its own package, so it still works, but
+        # we cannot support Debian Bullseye right now.
+        # - bullseye
```

Comment:
Is there an issue URL that we can reference here for posterity?

Reply:
There's no issue, it's just a part of Debian 11 being unstable.  The `*.deb` packages offered at [PowerShell/PowerShell](https://github.com/PowerShell/PowerShell) for Debian 11 were made in the past and some package they depend on has since been removed from or renamed in Debian 11.